src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -1290,12 +1295,12 @@ external dependencies to use in Flutter. A good place to find Flutter packages i
 
 ## Activities and fragments
 
-<aside class="alert alert-info" markdown="1">
-**Note:** You almost never want Android to restart the activity for a Flutter
-application. Especially since this goes directly against the advice of the Android
-documentation. So supporting split screen, for example, requires you to add
-`screenLayout` and probably `density` too.
-</aside>
+{{site.alert.note}}
```

Comment:
This section isn't right. Trying to prevent activities from restarting and banking on it will almost certainly cause bugs on Android. I'd just remove this section.